apim/3.x/README.adoc

```diff
@@ -208,7 +263,39 @@ pfx) |`+null+`
 
 See
 https://github.com/helm/charts/tree/master/stable/elasticsearch[Elasticsearch]
-for detailed documentation on optional requirements helm chart.
+for detailed documentation on optional requirements Helm chart.
+
+
+=== 4. Install The Gravitee Helm Chart
+
+To install the chart from the Helm repository with the release name
+`+graviteeio-apim3x+`, run:
+
+[source,bash]
+----
+$ helm install --name graviteeio-apim3x graviteeio/apim3
+----
+
+
+*Note:* If you're using Helm 3, this name parameter is no longer valid. Instead, please 
```

Comment:
Helm v3 is the current version (has been released on 2019).
So I think default commands should be compatible with helm v3 (without --name parameters).
And maybe add a compatibility note for older versions (or just add helm V3 as a prerequisite, at start of document)